internal/handlers/common/commands.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"context"
+
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+// Command represents a handler command.
+type Command struct {
+	Help    string
+	Handler func(Handler, context.Context, *wire.OpMsg) (*wire.OpMsg, error)
+}
+
+// Commands is a list of commands that common.Handler interface can support.
+var Commands = map[string]Command{
+	"listCommands": {
+		Help:    "Returns information about the currently supported commands.",
+		Handler: (Handler).MsgListCommands,
+	},
+
```

Comment:
listCommands is no longer a special case, no need to separate it with a blank line